src/fruity/fruity-host-session.vala

```diff
@@ -623,6 +623,7 @@ namespace Frida {
 				os["id"] = "ios";
 				os["name"] = properties.get_string ("ProductName");
 				os["version"] = properties.get_string ("ProductVersion");
+				os["type"] = properties.get_string ("ProductType");
```

Comment:
I'm thinking this belongs on the parent, as it's not a property of the operating system, but the system itself. Perhaps we should name it `model`, if that makes sense?

Reply:
Make sense. Updated!